shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-postgresql/src/main/java/org/apache/shardingsphere/proxy/frontend/postgresql/command/query/PostgreSQLCommand.java

```diff
@@ -51,6 +53,8 @@
     ROLLBACK(RollbackStatement.class),
     SET(SetStatement.class);
     
+    private static final Map<Class<? extends SQLStatement>, Optional<PostgreSQLCommand>> CACHE = new ConcurrentHashMap<>(16, 1);
```

Comment:
maybe XXX_CACHE is better ?

Reply:
@tristaZero @taojintianxia Rename to `COMPUTED_CLASSES` may be clearer.